### PR TITLE
prototype2-script: adapting the script to org workspaces

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: contrib/demo/prototype2-script/script [hunk 1]
```diff
@@ -66,17 +66,37 @@ pe "kubectl --kubeconfig ${CLUSTERS_DIR}/us-west1.kubeconfig apply -f ${DEMOS_DI
...
+c "Let's check that it is created and becoming ready."
+pe "kubectl get clusterworkspaces"
+
+c "We can switch over to the demo organization. For this first have to add it to our kubeconfig, with a little jq magic (this will soon become easier with the kcp kubectl plugin)."
+pe "kubectl config view -o json --raw > ${KCP_DATA_DIR}/.kcp/kubeconfig.json"
+pe "jq '.clusters += [.clusters[] | select(.name == \"system:admin\") | .name |= \"demo\" | .cluster.server += \"/clusters/root:demo\"]' < ${KCP_DATA_DIR}/.kcp/kubeconfig.json > ${KCP_DATA_DIR}/.kcp/admin.kubeconfig"
+pe "kubectl config get-clusters"
+c "Let's add a context for the demo organization, and switch to it."
+pe "kubectl config set-context demo --cluster demo --user admin"
+pe "kubectl config use-context demo"
```
Comment (sttts) [inline contrib/demo/prototype2-script/script:97]: What we want:
```
$ kubectl config use-context root
$ kubectl kcp workspace list
NAME     TYPE           PHASE
default  Organization   Ready
demo     Organization   Ready
```

**Comment by davidfestal** [comment]
:+1: for CLI plugin changes

File: contrib/demo/prototype2-script/script [hunk 2]
```diff
@@ -201,14 +218,14 @@ wait
 clear
 
 c "Now we're going to show our new workspaces APIs! Think of a workspace like your own Kubernetes apiserver, where you can create your own CRDs without worrying about conflicting with anyone else's."
-c "Let's start by creating a 'demo' workspace"
-pe "kubectl kcp workspace --token user-1-token create demo --use"
+c "We are in the demo workspace now. Let's create a new workspace called 'playground' as user-1 and switch to it."
+pe "kubectl kcp workspace --token user-1-token create playground --use"
```
Comment (davidfestal) [inline contrib/demo/prototype2-script/script:222]: Will this work with the last RBAC additions ? There should be the right permissions created for the demo org at the root level (maybe I missed where it is create...)

**Review by davidfestal — APPROVED** [review]

File: contrib/demo/prototype2-script/script (hunk 2 above)
Comment (sttts) [reply contrib/demo/prototype2-script/script:237]: it's fixed

**Pull request merged**